quipucords/api/management/commands/regenerate_missing_report_cache_files.py

```diff
@@ -0,0 +1,93 @@
+"""Regenerate missing DeploymentsReport cache files."""
+
+from django.core.management.base import BaseCommand
+
+from api.deployments_report.model import DeploymentsReport
+from utils.misc import file_exists
+
+
+class Command(BaseCommand):
+    """Django management command to regenerate missing DeploymentsReport cache files."""
+
+    help = "Regenerate missing DeploymentsReport cache files"
+
+    def write_info(self, message):
+        """Write info message to stdout."""
+        self.stdout.write(message)
+
+    def write_warning(self, message):
+        """Write warning message to stdout."""
+        self.stdout.write(self.style.WARNING(message))
+
+    def write_success(self, message):
+        """Write success message to stdout."""
+        self.stdout.write(self.style.success(message))
+
+    def find_reports_with_problematic_paths(self) -> set:
+        """Find reports with problematic paths and return their IDs."""
+        report_ids = set()
+        reports_paths = DeploymentsReport.objects.filter(
+            status=DeploymentsReport.STATUS_COMPLETE
+        ).values("id", "cached_fingerprints_file_path", "cached_csv_file_path")
+        self.write_info(
+            f"Found {len(reports_paths)} complete DeploymentsReports to check."
+        )
+        for report_paths in reports_paths:
+            if not report_paths["cached_fingerprints_file_path"]:
+                self.write_warning(
+                    "cached_fingerprints_file_path not set for DeploymentsReport "
+                    f"{report_paths["id"]}"
+                )
+            elif not file_exists(report_paths["cached_fingerprints_file_path"]):
+                self.write_warning(
+                    f"{report_paths["cached_fingerprints_file_path"]} is missing."
+                )
+                report_ids.add(report_paths["id"])
```

Comment:
Since that `if - elif` is pretty much the same, they just work on different keys, would it make sense to call them in a loop or extract to function?

Reply:
I do not remember what the else/elif here was, but it's gone now.